disnake/member.py

```diff
@@ -494,7 +494,7 @@ def discriminator(self) -> str:
         """:class:`str`:The user's discriminator.
 
         .. note::
-            This is being phased out by Discord; the username system is moving away from ``username#discriminator``
+            This is being phased out by Discord for non-bot users; the username system is moving away from ``username#discriminator``
```

Comment:
The official wording doesn't *really* sound like it?
`[...] application names and bot usernames will remain on the discriminator system.` and `In the future, we still plan to use application names as the identity for your app and bot across Discord.`, but then again also `Bot usernames will not change __immediately__, and will remain in the Username#1234 format with a discriminator.`

what would you suggest for that part of the docs?

Reply:
Well, they also have wording elsewhere that every user will be migrated at some point, which means we have to be as vague as possible and just link to the official documentation because they can't make up their mind.

Reply:
removed this change again for now, keeping it slightly more vague